recipes/autogluon.tabular/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "autogluon.tabular" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/autogluon/autogluon/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
All these autogluon packages have the same source and version number, right? Maybe it would be better to package them all with multiple outputs. It would save us some CI time. 1 download, 1 run, all the autogluon pakcages.


PS: I'm OK merging this for now but consider this in the future. The maintainer's life will be a bit easier too.

Reply:
@ocefpaf Thanks for the suggestion. This multiple outputs solution has been discussed at https://github.com/autogluon/autogluon/issues/612#issuecomment-1380873209. We plan to do this once all the dependencies and subpackages are available on conda-forge. Some of the dependencies are beyond our control, and we are not sure if we can make them all available on conda-forge. It can take several weeks before we can get to that points. For the time being, we prefer to go step by step as discussed at https://github.com/autogluon/autogluon/issues/612#issuecomment-1380226834